lib/launchers/amz.rb

```diff
@@ -615,6 +617,11 @@ def secret_key
       ec2.client.config.credentials.secret_access_key
     end
 
+    # @return [String]
+    def account_id
+      return @account_id
```

Comment:
Please change to `client.sts.get_caller_identity.to_h[:user_id]` , see what it returns in [1]. Or if by account id you mean `arn`, then use the `#arn` method directly. I'm not sure what exactly is needed because I can't see example of account ID you are using in the two PRs.

[1] https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/STS/Client.html#get_caller_identity-instance_method

Reply:
user_id is different from account_id... I don't know what is arn... OCM requires AWS account id (I'm not sure why) which is not included in any official AWS API library.

Reply:
So give an example of it. It would be surprising (or impossible) not to be in any API because if it is in no API, then how could OCM know that such account exists or not?

I expect it to be of the format of the method I linked above. But if it is not, we can further dig.

Reply:
After the chat, it seems like the body of this method can simply be 
```
    def account_id
      client.sts.get_caller_identity.to_h[:account]
    end
```
There is no need to ask users to provide this value. key and secret key are enough. Also this is a bug in OCM to require this value from the user. Should be reported IMO.

Reply:
FYI I also created an upstream issue https://github.com/openshift-online/ocm-cli/issues/216 to have it resolved for everybody in the future.

Reply:
@akostadinov that's awesome! I have just tried it and it works! Thanks!